sdk/eventhub/event-hubs/package.json

```diff
@@ -50,8 +45,8 @@
     "integration-test:browser": "karma start --single-run",
     "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json tsconfig.json src test samples --ext .ts --fix",
-    "lint": "eslint package.json tsconfig.json src test samples --ext .ts -f html -o event-hubs-lintReport.html || exit 0",
```

Comment:
Oh wait, should this be event-hubs-lintReport?

Reply:
Oh for sure! thank you!

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8644/commits/d691ab6b6882652f34b7b4f1f300deb650edb61d